dist/angular-patternfly.js

```diff
@@ -308,21 +308,38 @@ angular.module( 'patternfly.card' ).directive('pfAggregateStatusCard', function
 
  <file name="index.html">
    <div ng-controller="ChartCtrl">
-     <div pf-card head-title="Card Title" sub-title="Card Subtitle" filter="filterConfigHeader" style="width: 50%">
-       [Card Contents] <button>Click Me</button> Timeframe filter in header
+     <label class="label-title">Timeframe filter in header</label>
```

Comment:
The label element is intended for use in forms.  This would be better as a heading (e.g., h2).